lib/panda/error_handler.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'faraday'
+module FaradayMiddleware
+  class PandaErrorHandler < Faraday::Middleware
```

Comment:
Can we call it `Panda::ErrorMiddleware`? Using FaradayMiddleware module seems obsolete. Plus please rename file name to match the class name